Utils/Elasticsearch/mapping/tasks.mapping

```diff
@@ -184,6 +184,9 @@
               "ignore_above" : 256
             }
           }
+        },
+        "hs06": {
+          "type": "long",
         }
```

Comment:
Noticed a typo: here must be no comma in the end of `"type": "long"`, as it is the last member of `"hs06"`.
Will be fixed now...